src/renderer/store/modules/settings.js

```diff
@@ -235,6 +235,8 @@ const state = {
   rememberHistory: true,
   rememberSearchHistory: true,
   saveWatchedProgress: true,
+  // 'auto', 'semi-auto', 'never'
+  watchedProgressSavingMode: 'auto',
```

Comment:
As you've removed the old setting in various other places, please remove it here too.